org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/organizer/OrganizeColumnsForm.java

```diff
@@ -286,6 +309,11 @@ protected void execRowsSelected(List<? extends ITableRow> rows) {
             @Order(10)
             @ClassId("f607ab39-e616-4b27-b7c4-f6e437b6b1a3")
             public class ConfigNameColumn extends AbstractAlphanumericSortingStringColumn {
+              @Override
```

Comment:
Probably a matter of taste, but maybe add an empty line between class declaration and first method?